CMakeLists.txt

```diff
@@ -106,6 +106,14 @@ foreach(COMPUTE_ARCH IN LISTS K2_COMPUTE_ARCHS)
 endforeach()
 # ========= Settings for CUB end=========
 
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Modules)
+find_package(Valgrind)
+if(Valgrind_FOUND)
+  find_program(MEMORYCHECK_COMMAND ${Valgrind_EXECUTABLE})
```

Comment:
I think `Valgrind_EXECUTABLE` is a path, but `find_program`
https://github.com/k2-fsa/k2/blob/3dc0d7cb108bd9dbef522b45f7baeb7ebaa3f33c/cmake/Modules/FindValgrind.cmake#L30
requires a `name` or `NAMES` argument.

https://cmake.org/cmake/help/latest/command/find_program.html